libs/core/src/magento/models/outputs/index.ts

```diff
@@ -0,0 +1 @@
+export { MagentoMoney } from './money';
```

Comment:
let's use "requests" instead of "outputs". I was talking to Damien about this and he made some good points about the input-output model paradigm being a little confusing. Are they the outputs of a response or a transformer function or are they the outputs of daffodil toward a platform? It's a little more straightforward to just have them be requests and not requests instead of outputs and inputs.

Reply:
Wouldn't the inputs be the requests and the outputs be the not requests?

Reply:
changed